styles.css

```
@@ -0,0 +1,117 @@
+body {
+  background-color: burlywood;
+}
+.content {
+  display: flex;
+  flex-direction: column;
+  width: 680px;
+  padding: 24px 24px;
+  background-color: white;
+  font-family: sans-serif;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 16px;
+}
+.image {
+  height: 350px;
+  width: 680px;
+  border-radius: 16px;
+  margin-bottom: 10px;
+}
+h1 {
+  font-family: "young-serif";
+  color: hsl(24, 5%, 18%);
+  margin-bottom: 15px;
+}
+h3 {
+  font-family: "young-serif";
+  margin-left: 20px;
+  color: hsl(332, 51%, 32%);
+  font-size: bold;
+  margin-top: 0;
+  margin-bottom: 10px;
```

The same thing here. You can use the shorthand property for the `margin`.

```suggestion
  margin: 0 0 10px 20px;
```

---

Done